srcalc/src/main/java/gov/va/med/srcalc/web/controller/EnterVariablesController.java

```diff
@@ -68,6 +73,16 @@ public ModelAndView presentVariableEntry(
         // Present the view.
         final ModelAndView mav = new ModelAndView(Views.ENTER_VARIABLES);
         mav.addObject("calculation", workflow.getCalculation());
+        // In the case of using the "Return to Input Form" button, add the values already
+        // in the calculation to the variable entry object to maintain the previously calculated
+        // values on the entry page.
+        for(final Value value: workflow.getCalculation().getValues())
+        {
+        	final DynamicValueVisitor visitor = new DynamicValueVisitor(initialValues);
+        	visitor.visit(value);
+        	fLogger.debug("Key: {} Value: {}", value.getVariable().getKey(), value.getValue().toString());
```

Comment:
No need to call `toString()` here: let SLF4J do it for you if debug logging is enabled.